src/bag3_digital/layout/stdcells/inv_diff_chain.py

```diff
@@ -106,28 +106,26 @@ def draw_layout(self) -> None:
         self.add_pin('VSS', self.connect_wires(vss_list)[0])
 
         # connect chain
-        nodes = []
-        for i in range(1, length):
-            in_pin = drivers[i].get_pin('in')
-            inb_pin = drivers[i].get_pin('inb')
-            out_pin = drivers[i-1].get_pin('out')
-            outb_pin = drivers[i-1].get_pin('outb')
+        for idx in range(1, length):
+            in_pin = drivers[idx].get_pin('in')
+            inb_pin = drivers[idx].get_pin('inb')
+            out_pin = drivers[idx-1].get_pin('out')
+            outb_pin = drivers[idx-1].get_pin('outb')
             node = self.connect_wires([in_pin, out_pin])
             node_b = self.connect_wires([inb_pin, outb_pin])
-            nodes.append((node, node_b))
-            if label_nodes:
-                self.add_pin(f'mid<{i-1}>', node)
-                self.add_pin(f'midb<{i-1}>', node_b)
+            if export_nodes:
+                self.add_pin(f'mid<{idx-1}>', node)
+                self.add_pin(f'midb<{idx-1}>', node_b)
 
         # add input and output pins
```

Comment:
Pins cannot be reexported, only ports can be reexported. PyCharm should give a warning here. Please look at docstring.

Reply:
I don't understand why you want me to use reexport then. I have no port objects to re-export and the lines you are highlighting are new WIreArrays

Reply:
pins are always associated with ports. Look at inv_diff. Replace following 4 lines with:
self.reexport(drivers[0].get_port('in'))
self.reexport(drivers[0].get_port('inb'))
self.reexport(drivers[-1].get_port('out'))
self.reexport(drivers[-1].get_port('outb'))